futures-util/src/stream/mod.rs

```diff
@@ -126,8 +126,9 @@ cfg_target_has_atomic! {
     pub use self::futures_ordered::{futures_ordered, FuturesOrdered};
 
     #[cfg(feature = "alloc")]
-    mod futures_unordered;
+    pub mod futures_unordered;
```

Comment:
It was not possible to overwrite `FuturesUnordered::iter_pin_mut` and `FuturesUnordered::iter_mut` return types in the document simply by renaming on export, so I changed the module itself to be exposed like [`proc_macro2::token_stream`](https://docs.rs/proc-macro2/0.4/proc_macro2/token_stream/index.html).

Reply:
futures 0.1 also exposed `futures_unordered` module: https://docs.rs/futures/0.1.25/futures/stream/futures_unordered/index.html

Reply:
I overlooked the discussion in https://github.com/rust-lang-nursery/futures-rs/issues/793.